Mathlib/RingTheory/Localization/Defs.lean

```diff
@@ -8,7 +8,9 @@ import Mathlib.GroupTheory.MonoidLocalization.MonoidWithZero
 import Mathlib.RingTheory.OreLocalization.Ring
 import Mathlib.Tactic.ApplyFun
 import Mathlib.Tactic.Ring
+import Mathlib.Algebra.Ring.Pi
 import Mathlib.Algebra.BigOperators.Group.Finset
+import Mathlib.Algebra.Group.Submonoid.BigOperators
```

Comment:
Could you move these results to a new file, e.g. `Localization/Pi.lean`.

We shouldn't be adding new imports to `.Defs` files in order to add theorems!